azure-maven-plugin-lib/pom.xml

```diff
@@ -147,6 +147,11 @@
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.lingala.zip4j</groupId>
```

Comment:
Can we use the the original zip api in the JDK instead of adding a dependency here?

See: https://www.mkyong.com/java/how-to-compress-files-in-zip-format/

Reply:
Just compare the zip folder function, in the example #2 in the above link, we will have to write 50+ lines of code to zip a folder. And zip4j makes this happen in 5 lines of code.

Reply:
Adding a new third party dependency is fine. What worth concerning is that:
- Is this dependency a popular one?
- Is this dependency still actively releasing?
- How about the license issue?

For `net.lingala.zip4j`, What I'm concerning is that, I cannot find the source code of it, usage data from the Maven Central is not very high, and it has become stale since 2013. That's why I'm thinking if it is possible to control the logic by our own.